### PR TITLE
Fix Issue #6650 - Better documentation for Counter Flow vs Parallel flow coil parameter

### PR DESCRIPTION
**Pull request by dareumnam**
Fixed Issue #6650  

1. I-O Reference
- Added two sentences to point the user from the I-O reference sections at those field to the Engineering Reference sections
- Added informative sentences for CounterFlow and ParallelFlow

2. Engineering Reference
- Revised the wrong list of 7 heat exchanger model types

Pull request overview
---------------------
Please change this line to a description of the pull request, with useful supporting information including whether it is a new feature, or fixes a defect, a cross reference to any defects addressed in this PR, the type of changes to be made, and whether diffs are expected/justified based on this change.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add to input rules file for interfaces
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Required documentation updates?
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

File: doc/input-output-reference/src/overview/group-condenser-equipment.tex [hunk 1]
```diff
@@ -3956,15 +3956,17 @@ \subsubsection{Inputs}\label{inputs-15-002}
 \item
   \textbf{CrossFlowSupplyMixedDemandUnMixed}.~ Specifes a single-pass, cross-flow heat exchanger.~ The effectiveness will be calculated using a cross-flow heat exchanger correlation for flow mixed on the Loop Supply side and flow unmixed on the Loop Demand Side.
 \item
-  \textbf{CrossFlowSupplyUnMixedDemandMixed}.~~ Specifes a single-pass, cross-flow heat exchanger.~ The effectiveness will be calculated using a cross-flow heat exchanger correlation for flow unmixed on the Loop Supply side and flow mixed on the Loop Demand Side.
+  \textbf{CrossFlowSupplyUnMixedDemandMixed}.~ Specifes a single-pass, cross-flow heat exchanger.~ The effectiveness will be calculated using a cross-flow heat exchanger correlation for flow unmixed on the Loop Supply side and flow mixed on the Loop Demand Side.
```
Comment (Myoldmopar) [inline doc/input-output-reference/src/overview/group-condenser-equipment.tex]: @dareumnam can convert this to a description environment?

```
\begin{description}
  \item [CrossFlowWhatever] Description goes here.
  ...
\end{description}
```

This way you don't need to put periods at the end of each term name, no need for tilde ~ marks for spacing, etc.  Also, leave a space after the word and before you start the parentheses (like this).

**Review by Myoldmopar — CHANGES_REQUESTED** [review]
@dareumnam just a small requested change, let me know if there are any issues.

File: doc/input-output-reference/src/overview/group-condenser-equipment.tex (hunk 1 above)
Comment (dareumnam) [reply doc/input-output-reference/src/overview/group-condenser-equipment.tex]: @Myoldmopar Thank you for the review. I fixed the code as your suggestion. I am going to pull a request now.

File: doc/input-output-reference/src/overview/group-condenser-equipment.tex [hunk 2]
```diff
@@ -3948,22 +3948,17 @@ \subsubsection{Inputs}\label{inputs-15-002}
...
+\item [CrossFlowSupplyUnMixedDemandMixed] Specifes a single-pass, cross-flow heat exchanger. The effectiveness will be calculated using a cross-flow heat exchanger correlation for flow unmixed on the Loop Supply side and flow mixed on the Loop Demand Side.
+\item [CounterFlow] Specifies a counter-flow shell and tube heat exchanger (one fluid has the opposite direction to the other fluid). The effectiveness will be calculated using a counter-flow shell and tube heat exchanger correlation.
+\item [ParallelFlow] Specifies a parallel-flow shell and tube heat exchanger (both fluids have the same direction). The effectiveness will be calculated using a parallel-flow shell and tube heat exchanger correlation.
+\item [Ideal] Specifies an ideal heat exchanger. The effectiveness will be set to `1.0' and the specified UA will be ignored. The heat transfer rate will be calculated as the maximum possible heat transfer rate.
+\end{description}
```
Comment (Myoldmopar) [inline doc/input-output-reference/src/overview/group-condenser-equipment.tex:3959]: Much cleaner!  I'll build it now.

**Review by Myoldmopar — APPROVED** [review]
Looks much nicer!  Thanks!

image

**Pull request merged**